src/types/io.ts

```diff
@@ -41,9 +41,9 @@ export type PaginationParams<T = Record<string, never>> = {
 
 export type PaginationResult<T> = {
   items: T[];
-  nextCursor: string | undefined;
+  nextCursor?: string;
```

Comment:
by leaving the field empty rather than putting an undefined value, we can keep JSON parsing compatibility on stringified outputs